xml/geo_manage_hawk2_i.xml

```diff
@@ -0,0 +1,316 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect2
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<sect2 
+ xml:id="sec.ha.geo.manage.hawk2"
+ xmlns="http://docbook.org/ns/docbook" 
+ xmlns:xi="http://www.w3.org/2001/XInclude" 
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" >
+ <title>With the &haweb; (&hawk;2)</title>
+
+ <info>
+      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+        <dm:maintainer></dm:maintainer>
+        <dm:status>editing</dm:status>
+        <dm:deadline></dm:deadline>
+        <dm:priority></dm:priority>
+        <dm:translation></dm:translation>
+        <dm:languages></dm:languages>
+        <dm:release></dm:release>
+        <dm:repository></dm:repository>
+      </dm:docmanager>
+ </info>
+ &hawk-versions;
+ 
+  <sect3 xml:id="sec.ha.geo.manage.hawk2.dashboard">
+<!--taroth 2014-08-27: CAVE - the following is copied from ha_config_hawk.xml,
+ find a better solution next time-->
+   <title>Monitoring Multiple Clusters with the &hawk;2 Dashboard</title>
+   <para><remark>taroth 2015-10-29: krig: is the following still true?</remark>
```

Comment:
Partially true. The cluster information is no longer stored in a cookie, but on the server side. It is also synchronized between the nodes in the cluster if passwordless SSH access is correctly configured for root (same as is needed for the history explorer to work). This part is still true: The machine you are running hawk2 on does not even need to be part of any cluster, it can be a separate, unrelated cluster or even a machine that has hawk2 installed but is not part of any cluster. Such a offline hawk instance can also be used to view history reports.